core/variant/array.cpp

```diff
@@ -202,9 +202,10 @@ Error Array::resize(int p_new_size) {
 	return _p->array.resize(p_new_size);
 }
 
-void Array::insert(int p_pos, const Variant &p_value) {
-	ERR_FAIL_COND(!_p->typed.validate(p_value, "insert"));
+Error Array::insert(int p_pos, const Variant &p_value) {
+	ERR_FAIL_COND_V(!_p->typed.validate(p_value, "insert"), ERR_METHOD_NOT_FOUND);
 	_p->array.insert(p_pos, p_value);
```

Comment:
I looked at the code and I saw that Vector(which is Packed*Array but with type) may fail when using invalid position:
https://github.com/godotengine/godot/blob/2abe996414b8b551e69e29461de3ff1bcaf5a28f/core/templates/vector.h#L90
But in your code always is returned OK, even if insert fails

Reply:
I didn't catch that `Vector::insert` was returning, wiull use that value instead of hard-coding.